src/main/java/io/nats/client/support/NatsConstants.java

```diff
@@ -89,6 +89,7 @@ public interface NatsConstants {
     String INVALID_HEADER_COMPOSITION = "Invalid header composition";
     String INVALID_HEADER_STATUS_CODE = "Invalid header status code";
     String SERIALIZED_HEADER_CANNOT_BE_NULL_OR_EMPTY = "Serialized header cannot be null or empty.";
+    String OUTPUT_QUEUE_IS_FULL = "Output queue is full ";
```

Comment:
This string should probably be trimmed to not end in a space?

The space addition makes the previous:
`if (ex.getMessage().contains("Output queue is full"))`
To become:
`if (ex.getMessage().contains("Output queue is full "))`

Not sure, but this might have unintended side-effects?